cogboard-webapp/src/components/WidgetContent.js

```diff
@@ -3,11 +3,11 @@ import { object, string } from 'prop-types';
 
 import widgetTypes from './widgets';
 
-const WidgetContent = ({type, content}) =>  {
+const WidgetContent = ({ type, content }) =>  {
   const notFoundMessage = 'Widget type not found';
   const WidgetType = widgetTypes[type] ? widgetTypes[type].component : () => notFoundMessage;
 
-  return <WidgetType {...content} />
+  return <WidgetType { ...content } />;
```

Comment:
space after `{` and before `}`

Reply:
For props destructuring we could stick to no space formatting. Fixed.